src/sql/workbench/parts/query/browser/queryEditor.ts

```diff
@@ -56,148 +42,242 @@ export class QueryEditor extends BaseEditor {
+	private taskbar: Taskbar;
+	private splitview: SplitView;
 
-	private _sqlEditor: TextResourceEditor;
-	private _sqlEditorContainer: HTMLElement;
+	private inputDisposables: IDisposable[] = [];
 
-	private _taskbar: Taskbar;
-	private _taskbarContainer: HTMLElement;
-	private _listDatabasesActionItem: ListDatabasesActionItem;
+	private contextKey = new QueryEditorContext(this.contextKeyService);
 
+	private resultsVisible = false;
 
 	private queryEditorVisible: IContextKey<boolean>;
 
-	private _runQueryAction: RunQueryAction;
-	private _cancelQueryAction: CancelQueryAction;
-	private _toggleConnectDatabaseAction: ToggleConnectDatabaseAction;
-	private _changeConnectionAction: ConnectDatabaseAction;
-	private _listDatabasesAction: ListDatabasesAction;
-	private _estimatedQueryPlanAction: EstimatedQueryPlanAction;
-	private _actualQueryPlanAction: ActualQueryPlanAction;
+	//actions
+	private _runQueryAction: actions.RunQueryAction;
+	private _cancelQueryAction: actions.CancelQueryAction;
+	private _toggleConnectDatabaseAction: actions.ToggleConnectDatabaseAction;
+	private _changeConnectionAction: actions.ConnectDatabaseAction;
+	private _listDatabasesAction: actions.ListDatabasesAction;
+	private _estimatedQueryPlanAction: actions.EstimatedQueryPlanAction;
+	private _actualQueryPlanAction: actions.ActualQueryPlanAction;
+	private _listDatabasesActionItem: actions.ListDatabasesActionItem;
 
 	constructor(
-		@ITelemetryService _telemetryService: ITelemetryService,
+		@ITelemetryService telemetryService: ITelemetryService,
 		@IThemeService themeService: IThemeService,
-		@IInstantiationService private _instantiationService: IInstantiationService,
-		@IEditorService private _editorService: IEditorService,
-		@IQueryModelService private _queryModelService: IQueryModelService,
-		@IEditorDescriptorService private _editorDescriptorService: IEditorDescriptorService,
-		@IContextKeyService contextKeyService: IContextKeyService,
-		@IConnectionManagementService private _connectionManagementService: IConnectionManagementService,
-		@IConfigurationService private _configurationService: IConfigurationService,
-		@IStorageService storageService: IStorageService
+		@IStorageService storageService: IStorageService,
+		@IContextKeyService private readonly contextKeyService: IContextKeyService,
+		@IEditorService private readonly editorService: IEditorService,
+		@IInstantiationService private readonly instantiationService: IInstantiationService,
+		@IConfigurationService private readonly configurationService: IConfigurationService
 	) {
-		super(QueryEditor.ID, _telemetryService, themeService, storageService);
-
-		this._orientation = Orientation.HORIZONTAL;
+		super(QueryEditor.ID, telemetryService, themeService, storageService);
 
-		if (contextKeyService) {
-			this.queryEditorVisible = queryContext.QueryEditorVisibleContext.bindTo(contextKeyService);
-		}
+		this.queryEditorVisible = queryContext.QueryEditorVisibleContext.bindTo(contextKeyService);
+	}
```

Comment:
So there's already this queryContext that has this QueryEditorVisible property. Is there a reason you didn't either add more properties to this existing context or move the editor visible property into your new context? It seems to make more sense living with the others.

Reply:
The query editor context class was a remenant from a different change that didn't manage to make it into this PR, I'll remove all that code.